Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11
```

Comment:
I think we can use the slim image

Reply:
Yeah I had done some reading here https://hub.docker.com/_/python that suggests unless you are running on somewhere that has space constraints to use to default image. 

> Unless you are working in an environment where only the python image will be deployed and you have space constraints, we highly recommend using the default image of this repository.

Maybe we build a few of these in the future for for now going to leave since they highly recommend the default